src/effects/VST3/VST3Effect.cpp

```diff
@@ -341,10 +341,9 @@ bool VST3Effect::SupportsAutomation() const
 }
 
 bool VST3Effect::SaveSettings(
-   const EffectSettings &, CommandParameters & parms) const
+   const EffectSettings& settings, CommandParameters & parms) const
 {
-   if(mEditController == nullptr)
-      return false;
+   const VST3EffectSettings ownSettings = GetSettings(settings);
```

Comment:
Make `ownSettings` a reference and avoid a map copy

Reply:
ok